### PR TITLE
mapcss to init the clean up of covid19 tags

### PR DESCRIPTION
**Pull request by nlehuby**
To be activated in a few weeks, in France at least, to start the clean-up process of the `*:covid19` tags.

Questions:
* should I commit the generated python file from the mapcss too ?
* should I add `[inside("FR")]` on all tests to activate the checks only in France ?

### CODE REVIEW

**Comment by PanierAvide** [comment]
I agree on activating only in France these checks, and maybe we can ask teams from other countries if they are interested in such tooling ?

**Comment by frodrigo** [comment]
>     * should I commit the generated python file from the mapcss too ?

Yes

>     * should I add `[inside("FR")]` on all tests to activate the checks only in France ?

Yes

**Comment by nlehuby** [comment]
it's done

**Comment by frodrigo** [comment]
Pleas,e can you rename the plugin "Covid19.py" (capitalize).

I run it on a département, but return nothing.
```
time python ./osmose_run.py --skip-init --skip-frontend-check --no-clean --country=france_aquitaine_gironde --analyse=sax --plugin=covid19
```

File: plugins/covid19.validator.mapcss [hunk 1]
```diff
@@ -0,0 +1,74 @@
...
+
+*[opening_hours:covid19][opening_hours:covid19!~/same|off|open|restricted/][!opening_hours][inside("FR")]{
+    -osmoseItemClassLevel: "4010/202004/2";
+    throwWarning: tr("This store was on an adapted schedule during the lockdown. Are these opening hours still in effect ?");
+    fixChangeKey: "opening_hours:covid19=>opening_hours";
+
+    assertMatch: "way opening_hours:covid19=\"Mo-Su 09:00-20:00\"";
+    assertNoMatch: "way opening_hours:covid19=\"Mo-Su 09:00-20:00\" opening_hours=\"Mo-Su 09:00-20:00\"";
+    assertNoMatch: "way opening_hours:covid19=same";
+    assertNoMatch: "way opening_hours:covid19=restricted";
```
Comment (frodrigo) [inline plugins/covid19.validator.mapcss:73]: Use `-osmoseAssertMatchWithContext` and `-osmoseAssertNoMatchWithContext` to make assert pass the `[inside("FR")]`.

```css
-osmoseAssertMatchWithContext: list("node highway=milestone ref=A4 nat_ref=77PR38DC operator=SANEF", "inside=FR");
```

**Comment by nlehuby** [comment]
I got quite a few in Essonne

**Comment by frodrigo** [comment]
Thank you.

**Comment by frodrigo** [comment]
First results
http://osmose.openstreetmap.fr/fr/errors/?item=4010&class=202004

**Pull request merged**